### PR TITLE
Feat/section comment

### PR DESCRIPTION
**Pull request by drugstoreworker**
Issue #:
[Notion](https://www.notion.so/80ce8a70757c459a8ddf4333d3e0d43f?v=bd2c354608e249efa31cb5d6e2330cc1)

## PR type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe):

#### after:
Снимок экрана 2023-03-02 в 19 25 10

## Does this introduce a breaking change?

- [ ] Yes (please describe):
- [X] No

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Accessibility support has been tested
- [X] Responsive layout and functionality have been tested
  - [ ] desktop and mobile view responsiveness are supported
  - [ ] no loss of functionality or content when zoomed in to 400%
  - [ ] no horizontal scrolling when zoomed in to 400% (only vertical scrolling, exception: DataGrids)
- [ ] avoids using many levels of prop drilling
- [ ] check console on errors and warnings.
---

References:
- [Clean code](https://github.com/airbnb/javascript)

### CODE REVIEW

File: public/images/Vector 4.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="337" height="83" viewBox="0 0 337 83" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (AiteginJyldyzbekov) [inline public/images/Vector 4.svg:1]: you should delete this inneccessaty img

File: src/components/Card/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import StarsRating from "../StarsRating/StarsRating";
+import scss from "./Card.module.scss";
+export default function Card({ img, name, rating, desc, color }) {
+  console.log(img);
```
Comment (AiteginJyldyzbekov) [inline src/components/Card/Card.jsx:4]: you should delete this

**Comment by AiteginJyldyzbekov** [comment]
аватарки должны стоять по цетнру

**Comment by drugstoreworker** [comment]
created new

**Pull request closed without merging**